lib/components/tooltip.js

```diff
@@ -127,6 +134,38 @@ const addNpmLink = (data) => {
   return npm;
 };
 
+const addAtomLink = (data) => {
+  if (!data || !data.name) {
+    return undefined;
+  }
+
+  const dirlink = new Directory(atom.workspace.getActiveTextEditor().buffer.file.getParent().path+"/node_modules/"+data.name)
+  const existDirlink = dirlink.existsSync()
```

Comment:
Looks like `;` is missing, run `npm run lint` to fix linting issues. Just to keep the code consistent ;) This is actually my bad, I hadn't linked any `ci` into the pull request in github. I'll do it shortly.

Reply:
As a standardjs user, I always forget the semicolons along the way 😅